src/node/test/channels.cpp

```diff
@@ -21,7 +21,8 @@
 namespace ccf::enclavetime
 {
   std::atomic<long long>* host_time_us = nullptr;
-  std::chrono::microseconds last_value(0);
+  std::atomic<std::chrono::microseconds> last_value(
```

Comment:
This is due to the new warning of data race after fixing the origin `consensus` warning.